### PR TITLE
Added new tab to Timelog component. Displaying the last three summari…

### PR DESCRIPTION
**Pull request by LeeSingChanYau**
It was requested that we have a new tab in the Timelog Component in the dashboard to display the last three summaries of the current user we are seeing. This PR adds a new tab to the timelog and also displays the last three summaries. I tested this and it is displaying the last three summaries of the userProfile object that is returned, but I encountered a problem. The last three summaries I am getting are all of the current week (8th of January 2023 deadline), I tried different accounts and I see the same problem. 
Files changed:
- Timelog.jsx
- Dashboard.jsx
- SummaryBar.jsx

Filex created:
- WeeklySummaries.jsx

### CODE REVIEW

**Comment by rauleffting** [comment]
Hi @LeeSingChanYau. I think there might be a problem in that part of the code. Probably on loading the HTML elements as a string:

211335428-2cd6311c-dfee-4aaa-bb74-6c3fe4c09ab7

Could you take a look? I think it's very simple to fix. Thanks!

**Comment by agujjar2206** [comment]
I believe these are the last three summaries of the userProfile object.

**Comment by suigunfjc** [comment]
Can't open the dashborad page, there is an error in `<WeeklySummaries>` component.
Screenshot 2023-02-18 191326

File: src/components/Timelog/WeeklySummaries.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        <div>
+          <h4>This week's summary</h4>
+          <h4>{userProfile.weeklySummaries[0].summary}</h4>
+        </div>
+      ) : (
+        <h4>
+          {userProfile.firstName} {userProfile.lastName} did not submit a summary for this week.
+        </h4>
+      )}
+      {userProfile.weeklySummaries[1].summary ? (
```
Comment (suigunfjc) [inline src/components/Timelog/WeeklySummaries.jsx:16]: For a user don't have any summary, the length of `userProfile.weeklySummaries` is 1 and line 16&26 will cause a crash.

**Review by suigunfjc — DISMISSED** [review]
Hi @LeeSingChanYau , please address my comment on WeeklySummaries.jsx, that is causing me not able to launch the branch.

**Comment by NyahJ** [comment]
Hi  @LeeSingChanYau,

After using different testing accounts, the result on my side went well greatly:
 
截屏2023-02-18 下午9 48 30

截屏2023-02-18 下午9 49 01

A tiny advice on the font of the last 3 summaries, it would be better to be smaller and consistent with the page :)

**Review by tingedudu8 — CHANGES_REQUESTED** [review]
Hi @LeeSingChanYau, I tested your PR and the weekly summaries are not properly displayed. Please check your rendering part of code.
PR639

**Review by vitorsaa2k — CHANGES_REQUESTED** [review]
Hi @LeeSingChanYau,

It worked in some parts but not in others.
image

- The rendering of the summaries seems to not be working well.
- Also, would be good if the title that separate the date of the summaries was more visable, Maybe changing the color or adding something around the make obvius that it is a title

**Comment by ayush0904** [comment]
Hi @LeeSingChanYau Please see the below image.
The HTML tags are also getting printed along with the text.
Screen Shot 2023-03-04 at 2 14 10 PM

**Comment by mpavithr** [comment]
Hi, as other reviewers have noted above, I am also facing the same issue with the tags being printed.
pr639_1

**Comment by LeeSingChanYau** [comment]
@vitorsaa2k @mpavithr @ayush0904 Hello guys, thank you for your feedback. Please pull the changes, and review again. You shouldn't see the html tags anymore. Thanks!

**Review by vitorsaa2k — APPROVED** [review]
Working as intended.
image

**Comment by one-community** [comment]
Excellent, thank you everyone. Moving to final review.

**Review by rauleffting — APPROVED** [review]
Hi, @LeeSingChanYau. It worked perfectly.

image

Thank you!

**Pull request merged**